src/challenge.ts

```diff
@@ -39,7 +39,9 @@ function addReview(
   comment: string
 ): ReviewedBook {
   // write your code here...
-
+  book.reviews
+    ? book.reviews.push({ reviewer: reviewer, comment: comment })
+    : (book.reviews = [{ reviewer: reviewer, comment: comment }]);
```

Comment:
Ternary operators should only really be used as _expressions_ (where the entire ternary operator will be replaced with a value. They should not have side-effects or be used as a general replacement for if statements.

A better ternary use here would be:
```TS
  const newReview: Review = { reviewer, comment };

  book.reviews =
    book.reviews ? [...book.reviews, newReview] : [newReview];
 ```